### DISCUSSION TITLE
Welcome to foocars Discussions!

### DISCUSSION THREAD

**Discussion by ricklon** [fubarlabs/foocars#58] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lcipolina** [comment]
Hi! I am Lucia and I would like to keep contributing with foocars. Albeit, remotely for now.

I really like the project and hope it continues!

**Comment by ricklon** [comment]
Yay. I'm glad you posted.

I've got new circuit boards for the project and working on some new updates
for a highschool course coming up soon. One area I'm looking at is making
this project compatible with the Donkey Car Virtual car simulator. This
could go in some really fun directions.

--Rick

On Thu, Jan 7, 2021 at 12:09 PM Lucia Cipolina Kun <notifications@github.com>
wrote:

> Hi! I am Lucia and I would like to keep contributing with foocars. Albeit,
> remotely for now.
>
> I really like the project and hope it continues!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fubarlabs/foocars/discussions/58#discussioncomment-267466>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABV3ZAKOOJXPBLIQY4QDFDSYXTGNANCNFSM4VZCC3TA>
> .
>


-- 
Co-founder
Fair Use Building and Research (FUBAR) Labs
http://fubarlabs.org

**Reply by lcipolina** [reply]
oh nice! I will check that simulator out! seems interesting 👍
